docs/symbols-offKeyValue.md

```diff
@@ -0,0 +1,7 @@
+@property {Symbol, Function} can-symbol/symbols/offKeyValue can.offKeyValue
+@parent can-symbol/symbols/observe
+@description A symbol placed on a constructor to reference a function used to stop listening to changes on a specific named property.
+
+@signature `Example` `obj[canSymbol('can.offKeyValue')] = function(key, fn) { this.removeEventListener(key, fn) };`
```

Comment:
does this show up right?  I don't know what this will end up looking like. afaik bit-docs doesn't support this.